### DISCUSSION TITLE
Performance breakdown for various segments of Marathon maps

### DISCUSSION THREAD

**Discussion by vipotaenko02** [ppy/osu#21233] [Ideas]
When playing long Marathon maps, I always get curious on where and how did I perform. So I thought it would be great if osu! can show not only my final accuracy, but also accuracy of particular song/segment that is being the part of a map. It can also show influence of each individual segment on final accuracy (e.g. subtraction of "Final accuracy after X segment" and "Final accuracy after X-1 segment") and comparsion of each individual segment's accuracy with its accuracy on my top score on this map. Performance Breakdown would be a great place to show this kind of information. Each map's segments can be added in Editor in Timing section, so mappers can just name every segment and give it a start time (start time of X segment will be the end of X-1 segment).

**Comment by Doksha** [comment]
It would probably be easier to implement something like the health graph, multiplying a bellcurve to an array of judgements accross the whole map (over having to define sections, although breaks could be an indicator if its implemented like that, but break spam is a thing so likely wont work).
or simply take the exact accuracy at each point in time and use that, adjusting the minimum value to the lowest accuracy reached after like 10 or 20 hitobjects.
